Makefile

```diff
@@ -28,6 +21,10 @@ test-lib-foreach:
 check-test-lib: $(TEST_LIB_TESTS)
 
 test: check
+	TESTED_SCENARIO=test tests/remote-containers.sh
+
+test-openshift-4: check
```

Comment:
Shouldn't be this targer added to .PHONY ones? Probably also shellcheck,  test-lib-foreach  (this one probably should not exist at all), pre-commit-check, check-squash, check-latest-imagestream, check-betka targets should be marked as phony.

Reply:
.PHONY added.